pymc_marketing/mmm/delayed_saturated_mmm.py

```diff
@@ -1146,3 +1151,100 @@ def sample_posterior_predictive(
             )
 
         return posterior_predictive_samples
+
+    def add_lift_test_measurements(
+        self,
+        df_lift_test: pd.DataFrame,
+        dist=pm.Gamma,
+        name: str = "lift_measurements",
+    ) -> None:
+        """Add lift tests to the model.
+
+        Parameters
+        ----------
+        df_lift_test : pd.DataFrame
+            DataFrame containing the lift test measurements.
+        dist : pm.Distribution, optional
+            The distribution to use for the likelihood, by default pm.Gamma
+        name : str, optional
+            The name of the likelihood, by default "lift_measurements"
+
+        Raises
+        ------
+        RuntimeError
+            If the model has not been built yet.
+        KeyError
+            If the 'channel' column is not present in df_lift_test.
+
+        Examples
+        --------
+        Build the model first then add lift test measurements.
+
+        .. code-block:: python
+
+            model = DelayedSaturatedMMM(
+                date_column="date_week",
+                channel_columns=["x1", "x2"],
+                control_columns=[
+                    "event_1",
+                    "event_2",
+                ],
+                adstock_max_lag=8,
+                yearly_seasonality=2,
+            )
+
+            X: pd.DataFrame = ...
+            y: np.ndarray = ...
+
+            model.build_model(X, y)
+
+            df_lift_test = pd.DataFrame({
+                "channel": ["x1", "x1"],
+                "x": [1, 1],
+                "delta_x": [0.1, 0.2],
+                "delta_y": [0.1, 0.1],
+                "confidence": [0.1, 0.1],
```

Comment:
A few side notes:

1. Confidence should be the `sigma` define for an experiment, no?
2. Tools like Causal Impact (Example) provide the sigma around the mean estimate, and ideally, this value should be equal to the standard deviation given by the experiment framework, right?
3. What criteria are used to define confidence? In case a tool like `causalpy` doesn't deliver it to you, what are possible frameworks or recommended values?

It's important to clarify the definition of confidence and demonstrate how having different degrees of reliability can impact the results of an experiment. I think that marketers are not very familiar with calibration and those questions may arise!

Reply:
Thanks for taking a look @carlosagostini 

1. Yes, correct. I changed the name to `sigma` from `confidence`. I think that would be more clear
2. Do you have a link for this example?
3. Good point to point out any conversions needed or 

I have started a notebook

Reply:
Causal impact example output:

```python
## Posterior inference {CausalImpact}
## 
##                          Average        Cumulative  
## Actual                   117            3511        
## Prediction (s.d.)        107 (0.37)     3196 (11.03)
## 95% CI                   [106, 107]     [3174, 3217]
##                                                     
## Absolute effect (s.d.)   11 (0.37)      316 (11.03) 
## 95% CI                   [9.8, 11]      [294.9, 337]
##                                                     
## Relative effect (s.d.)   9.9% (0.35%)   9.9% (0.35%)
## 95% CI                   [9.2%, 11%]    [9.2%, 11%] 
## 
## Posterior tail-area probability p:   0.001
## Posterior prob. of a causal effect:  99.9%
```